### DISCUSSION TITLE
Welcome to Dev Cheats Discussions!

### DISCUSSION THREAD

**Discussion by excalith** [excalith/dev-cheats#2] [Announcements]
## 👋 Welcome to Dev Cheats Discussions!

Dev Cheats is a community-driven project aimed at providing developers of all skill levels with a convenient reference for various command line tools and their usage examples. We are excited to have you join us on this journey!

This Discussions space is intended to be a place where community members can connect, share ideas, ask questions, and engage with each other. We encourage you to be welcoming and open-minded, as this is a community that we are building together 💪.
Categories

To help keep the discussions organized, we have created the following categories:

- **Announcements:** This category will contain important announcements about Dev Cheats.
- **General:** General discussions about anything related to Dev Cheats or programming in general.
- **Ideas:** A category to discuss new features and enhancement ideas for Dev Cheats.
- **Polls:** Polls about the general direction of the Dev Cheats project and its features.
- **Q&A:** A place to ask questions and get help regarding the Dev Cheats project.
- **Success Stories:** Share your inspiring stories about how Dev Cheats has helped you in your journey. 

We are excited to hear your thoughts, feedback, and ideas about Dev Cheats! Let's make this project a valuable resource for developers around the world.

**Comment by abdallahbouhannache** [comment]
Hello and good luck with your idea .
But i wanna know what is the difference between your work and this prexisiting one 
https://devdocs.io/
Are you going to make it somehow like ai assisted ?or something like with exemples lile codepilot or do you think you have a different strategy for your work!
Best of luck .

**Reply by excalith** [reply]
Hello @abdallahbouhannache, thanks for your question! I think it would be great to add to readme about the key differences.

The key difference of Dev Cheats is to make it more accessible to all developers with any skill level. The philosophy behind this project is, you might be an experienced terminal junkie or a new developer want to get into tools with confusing arguments, Dev Cheats should appeal to you. Since the apps out there either shows everything (which might be overwhelming for some) or shows less information which may not be the result you are looking for. With Dev Cheats, I want to focus on this problem.

Another difference I think that applies here is to make Dev Cheats more community-driven, so it can shape over the time by the community requests, making it more intuitive for all. If Dev Cheats could get enough attention and love from developers, I believe it would be a great place to share knowledge of tools and usages in an opinionated and friendly way.

About the AI though, I actually I have thought of implementing it for some "how to" queries for a long time, but there are bunch of AI tools out there already doing it perfectly. However, as I mentioned above, this might be implemented later if that is what community asks for!

Thanks again for this awesome question, pushed me towards writing down a manifest!
